<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeamformerSettingsViewWidget</class>
 <widget class="QWidget" name="BeamformerSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="m_label_timePoint">
       <property name="text">
        <string>Time point (incl. pre stim):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_label_triggerType">
       <property name="text">
        <string>Trigger type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_label_method">
       <property name="text">
        <string>Weight normalization method:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_comboBox_method">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_comboBox_triggerType"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_spinBox_timepoint">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
